sdk/src/main/java/io/customer/sdk/data/store/DeviceStore.kt

```diff
@@ -47,4 +50,15 @@ internal class DeviceStoreImp(
             append(" $customerAppName/$customerAppVersion")
         }
     }
+
+    override fun buildDeviceAttributes(): Map<String, Any> {
+        return mapOf(
+            "device_os" to deviceOSVersion,
+            "device_model" to deviceModel,
+            "app_version" to customerAppVersion,
+            "cio_sdk_version" to customerIOVersion,
+            "device_locale" to deviceLocale,
+            "push_subscribed" to true
```

Comment:
Is there a reason that this is always `true`? 

[I remember talking about what this attribute means as a squad](https://www.notion.so/custio/504e69222e0e43f3982f51e80aaaf7b3?v=bcaa50a92c5b47d6920bc9ee8ff5049a). We decided (for now) that on Android, this attribute would be true if at least 1 Android push channel was enabled. Is that possible for us to query?